src/get-rollup-options.js

```diff
@@ -202,6 +202,21 @@ export default function(options, format) {
     external = [external]
   }
 
+  let globals
+  if (options.globals) {
+    globals = options.globals
+    if (typeof globals === 'string') {
+      globals = {}
+      options.globals.split(',').forEach(g => {
+        const s = g.split(':')
```

Comment:
maybe add note about such usage to the doc as well?

Reply:
It is already added to cli usage. But i can add it to docs too :)

Reply:
I'm not sure if it's necessary though, you can in fact use sth like `--global.foo foo --global.bar bar`